terraform/applications.tf

```diff
@@ -49,12 +50,12 @@ module "k8s" {
 }
 
 module "k8s_worker" {
-  source      = "git::https://github.com/canonical/k8s-operator//charms/worker/terraform?ref=main"
+  source      = "git::https://github.com/canonical/k8s-operator//charms/worker/terraform?ref=charmed-etcd"
```

Comment:
should be able to revert this now that the following is merged
* https://github.com/canonical/k8s-operator/pull/747